### PR TITLE
Update ADA doc to show alpha release of Datadog provider

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes https://github.com/pipe-cd/pipe/issues/1143

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs/operator-manual/piped/adding-an-analysis-provider.md [hunk 1]
```diff
@@ -7,9 +7,17 @@ description: >
 ---
 
 
-You define the information needed to connect from your Piped to the Analysis Provider.
+To enable [Automated deployment analysis](/docs/user-guide/automated-deployment-analysis/) feature, you have to set the information needed to connect from your Piped to the [Analysis Providers](/docs/concepts/#analysis-provider).
```
Comment (nghialv) [inline docs/content/en/docs/operator-manual/piped/adding-an-analysis-provider.md:11]: nit: `, you have to set the needed information for Piped to connect to the Analysis Provider.`

Comment (nakabonne) [reply docs/content/en/docs/operator-manual/piped/adding-an-analysis-provider.md:11]: Looks better 👍

**Comment by nakabonne** [comment]
Fixed done.

**Pull request merged**